pkg/services/navtree/navtreeimpl/admin.go

```diff
@@ -128,7 +128,7 @@ func (s *ServiceImpl) getAdminNode(c *contextmodel.ReqContext) (*navtree.NavLink
 		})
 	}
 
-	if hasAccess(ac.ReqGrafanaAdmin, ac.EvalPermission(ac.ActionSettingsRead)) && s.features.IsEnabled(featuremgmt.FlagStorage) {
+	if hasAccess(ac.ReqGrafanaAdmin, ac.EvalPermission(ac.ActionSettingsRead, ac.ScopeSettingsAll)) && s.features.IsEnabled(featuremgmt.FlagStorage) {
```

Comment:
Not entirely sure what the flag storage is - however since with the legacy access control the requirement was that only Server Admin's can access it, I assume it should be true also with the new RBAC system. With that assumption, I've decided to add the scope here too.